dbt/include/snowflake/macros/adapters.sql

```diff
@@ -72,14 +72,16 @@
 )
 {% endmacro %}
 
-{% macro snowflake__create_view_as(relation, sql) -%}
+{% macro snowflake__create_view_as_with_temp_flag(relation, sql, temporary=False) -%}
```

Comment:
Feel like I'm missing something if `temporary` arg defaults to `False` and we don't provide this value in `snowflake__create_view_as` won't the if condition on 83 never be satisfied? Should this be sourced from the config?